fake-server/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const bodyParser = require("body-parser");
+const app = express();
+const port = 3000
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.post('/', (req, res) => {
+    console.log(req.body);
+    let { username, password } = req.body || {};
+    if ((username === "james") && (password = "Password1!")) {
+        res.status(201).send("Login succeeded");
+        console.log("Login succeeded for", username);
+    } else {
+        res.status(403).send("Login failed");
+        console.log("Login failed for", username);
+    }
+});
+
+app.listen(port, () => console.log(`Fake login server listening at http://localhost:${port}`));
```

Comment:
In reality we'd want to set this server up to use `https` rather than `http`

Reply:
agreed. I think this is fine for dev work though - a real server would need to be configurable using environment variables or the like

Reply:
Indeed 👍